src/SFA.DAS.ProviderApprenticeshipsService.Web/Orchestrators/BulkUploadOrchestrator.cs

```diff
@@ -156,14 +158,15 @@ private async Task<IEnumerable<UploadError>> GetOverlapErrors(IList<Apprenticesh
             var overlapResponse = await Mediator.Send(overlapRequest);
 
             if (overlapResponse.Overlaps.Any())
-            {
-                
+            {                
                 var validationErrors = uploadedApprenticeships.ToList();
 
                 foreach (var overlapGroup in overlapResponse.Overlaps)
                 {
                     foreach (var overlap in overlapGroup.OverlappingApprenticeships.ToList())
                     {
+                        if (string.IsNullOrEmpty(overlap.Apprenticeship.ULN)) break;
```

Comment:
should this be `continue` rather than `break`?

Reply:
Yes, it should be continue , resolved